code/src/frontend/styles/main.scss

```diff
@@ -389,9 +411,35 @@
         }
       }
     }
+
+    &--light-minimal {
+      background: #fff;
+      color: #190000;
+    }
+
+    &--dark-minimal {
+      background: #292a2d;
+      color: #fff;
+
+      .inquiry-popup {
+        &__close {
+          color: #fff;
+          transition: color ease 0.2s;
+
+          &:hover,
+          &:focus-visible {
+            color: #bbb;
+          }
+        }
+      }
+    }
   }
 
   &-hidden {
     display: none;
   }
 }
+
+.button {
+  margin: 30px 0 0;
```

Comment:
Should this class be nested inside .inquiry class? .button is a quite generic class name, and since this stylesheet gets added at the head end, it could unintentionally overwrite the margin values of an existing .button class.

Reply:
done and done